app/javascript/app/document-capture/image_upload.js

```diff
@@ -0,0 +1,34 @@
+import useCSRF from './hooks/use-csrf';
+
+const imageUploadURI = '/api/verify/upload';
+
+function uploadImages(frontImage, backImage, selfieImage) {
+  const csrfToken = useCSRF();
+  return new Promise((resolve, reject) => {
+    fetch(imageUploadURI, {
```

Comment:
Noting that `window.fetch` isn't supported in Internet Explorer, and as far as I could tell, isn't yet polyfilled.

We could choose to try to find a polyfill to use, or use the old-fashioned [`XMLHttpRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) instead.

Reply:
> Noting that `window.fetch` isn't supported in Internet Explorer, and as far as I could tell, isn't yet polyfilled.
> 
> We could choose to try to find a polyfill to use, or use the old-fashioned [`XMLHttpRequest`](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) instead.

This is still an issue, but extends beyond the scope of this pull request, as there's already a few issues with Internet Explorer needing a polyfill.

Tracking at: LG-3241